src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -862,6 +921,45 @@ private void DeleteValue(TransactionOperationContext context, string type, Blitt
             }
         }
 
+        private void DeleteCertificate(TransactionOperationContext context, string type, BlittableJsonReaderObject cmd, long index)
+        {
+            try
+            {
+                var certs = context.Transaction.InnerTransaction.OpenTable(CertificatesSchema, CertificatesSlice);
+                var command = (DeleteCertificateFromClusterCommand)CommandBase.CreateFrom(cmd);
+
+                using (Slice.From(context.Allocator, command.Name, out Slice keySlice))
```

Comment:
why aren't you calling `DeleteCertificate` here?

Reply:
fixed